flintrock/flintrock.py

```diff
@@ -172,8 +175,11 @@ def cli(cli_context, config, provider):
 @click.option('--spark-version',
               help="Spark release version to install.")
 @click.option('--spark-git-commit',
-              help="Git commit hash to build Spark from. "
-                   "--spark-version and --spark-git-commit are mutually exclusive.")
+              help="Git commit hash to build Spark from."
+                   "Can be \"latest\" to build Spark from the latest available commit."
+                   "--spark-version and --spark-git-commit are mutually exclusive.",
+              default="latest",
```

Comment:
Nitpick: Flintrock follows the pattern of single quotes for programmatically meaningful values, and double quotes for English prose. So throughout this PR, I want us to use `'latest'` instead of `"latest"`.

The exception to this is when naming command-line options, to follow the convention set there by Click.